### PR TITLE
Change widget inits to Fliplet.Widget.instance

### PR DESCRIPTION
**Pull request by YaroslavOvdii**
@sofiiakvasnevska 
## Issue
https://github.com/Fliplet/fliplet-studio/issues/5354

## Description
Change widget inits to Fliplet.Widget.instance

## Backward compatibility

This change is fully backward compatible.

### CODE REVIEW

File: js/build.js [hunk 1]
```diff
@@ -1,24 +1,31 @@
-$('.linked[data-list-item-id]').click(function (event) {
-  event.preventDefault();
+(function () {
```
Comment (squallstar) [inline js/build.js:1]: any reason for having to wrap this into a self executing function?

Comment (YaroslavOvdii) [reply js/build.js:1]: @squallstar 
I've done it as a recommendation from this [comment](https://github.com/Fliplet/fliplet-studio/issues/5354#issuecomment-564932978).

Should I remove it?

**Review by squallstar — APPROVED** [review]
@sofiiakvasnevska this is ready to be deployed on **staging** for QA.

- **Risk**: Medium
- **Affected**: Studio users configuring the component
- **Verify**: Verify the component is render correctly and is also refreshed once changed are made

File: js/build.js (hunk 1 above)
Comment (squallstar) [reply js/build.js:1]: I see, it's all good then.

**Pull request merged**